lib/movieField.ts

```diff
@@ -0,0 +1,15 @@
+export type MovieField = {
```

Comment:
suggestion: you could have a file called `lib/types` that contains all the types related to the movies :)

Reply:
and this one too